build/es5.umd/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Yes, we can definitely make it simpler. It was a demo on how we could build as much variants as we want.

We'll still need at last one tsconfig per variant.